### PR TITLE
Create CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?

Finally creating a CONTRIBUTING.md file for all Store Framework repos 🙌🏻 .

Here you can check the rendered version: https://github.com/vtex-apps/store-discussion/blob/feat/contributing/CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+3. Run `yarn` inside the `/react` and/or `/node` folders. This will install the necessary dependencies for the app itself.
+4. Make your changes.
+5. Ensure that the test suite passes. Run `yarn test` in the repository root to run all tests in the app. In some apps, running this command in the root will give you an error saying that there's no such command. In this case, run `yarn test` inside the `/react` or `/node` folders.
+6. Make sure your code is formatted properly by running `yarn lint` in the repository root.
+7. Update the `CHANGELOG.md` file. Our changelog files are written following the pattern described at [https://keepachangelog.com/en/1.0.0/](https://keepachangelog.com/en/1.0.0/).
+
+    To make your PR easier to merge and avoid getting notified by conflicts every time a change to the changelog is made, make sure to place your entry**directly** below the `[Unreleased]` header, with no empty line in between.Here's an example:
```
Comment (lariciamota) [inline CONTRIBUTING.md:52]:
```suggestion
    To make your PR easier to merge and avoid getting notified by conflicts every time a change to the changelog is made, make sure to place your entry **directly** below the `[Unreleased]` header, with no empty line in between. Here's an example:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+If you spot any outdated or wrong information in any of our docs, please feel free to open a pull request to update it, no issue needs to be created in this case.
+
+## Store Framework Office Hours
+
+Office Hours is a weekly one-hour meeting with community members, the core team, and other VTEXers. It's a great opportunity to talk to us, get help, and give us some feedback about the framework.
+
+The event is open to everyone and you are always welcome! Important to note that the event is conducted in Portuguese, but if you don't speak the language, no worries. You can come and ask us your questions in English.
+
+To participate in the event, join [this Zoom room](https://vtex.zoom.us/j/96795350497?pwd%3DSzBYY25FdnQrSkdwbWtKQVh3K2lWdz09&sa=D) at Tuesdays 4PM GMT-3 (Brasília time).
```
Comment (lariciamota) [inline CONTRIBUTING.md:85]:
```suggestion
To participate in the event, join [this Zoom room](https://vtex.zoom.us/j/96795350497?pwd%3DSzBYY25FdnQrSkdwbWtKQVh3K2lWdz09&sa=D) on Tuesdays at 4PM GMT-3 (Brasília time).
```

**Review by lariciamota — APPROVED** [review]
Nice documentation! 👏

**Review by iaronaraujo — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+This is the single contributing guide to help you contribute to any of Store Framework's repositories. Keep in mind that all contributions made in Store Framework repositories are open to the public. And the pull request + code review cycle is the same for core team members, engineers inside of VTEX, and external contributors.
+
+One important point to make here is that we want our community to be as open and accessible as possible, and language should not be a barrier. Throughout this guide, we'll recommend you to open issues and pull requests in our repositories, both of which will include a writing step. If you're not comfortable writing in English, please feel free to do so in Portuguese.
+
+## Bugs
+
+If you've found a bug or something is not behaving in the way you expected it to, feel free to [open an issue](https://github.com/vtex-apps/store-discussion/issues/new/choose) **after** you've checked to see if anyone else already reported the same bug. This is important to make it easier for us to track these issues. We'll do our best to review them and let you know when we're working on a fix.
```
[the remark this answers is not in the document]
Comment (victorhmp) [reply CONTRIBUTING.md:9]: That's great! 😄

**Pull request merged**